bconsole: history repeated entries (#1162)
bareos/bareos: CHANGELOG.md

- `bvfs_update` now uses `unordered_map` instead of `htable` for the pathid cache [PR #1138]
- cats: filtered zero file jobs list is now sorted [PR #1172]
- dird: console: changed list jobs jobstatus argument to accept comma separated value [PR #1169]

> - bconsole: command is now only added once to history when run multiple times in a row [PR #1162]

Reviewer: ```suggestion
- bconsole: multiple identical commands are only added to history once [PR #1162]
```
